### DISCUSSION TITLE
ed080xc1 The right side of the panel shows a problem

### DISCUSSION THREAD

**Discussion by hongchenfeng** [vroland/epdiy#384] [Q&A]
![IMG20241208110035.jpg](https://github.com/user-attachments/assets/309cd1fd-0050-4ae6-ab16-5c926f3d6599)

photo showed.
panel has worked but the right side has problem. what happened. i think it's the horizontal timing and check it, inchulding     line_front_porch, end_line.
The right side of panel duplicate from the   left side, and indicates the vertical timing  is right. And how to correct it, help me please!

**Comment by martinberlin** [comment]
Hello @hongchenfeng 
Did you try using another waveform like ED097TC2?
I don’t know why but that solved a similar issue for me

**Reply by hongchenfeng** [reply]
yes，had tryied but worked. and will waveform lead to this
 problem ? it's a question about the left side working well.
esp32 is not clear about driving LCD, i plan to drive it with FPGA next step.

**Reply by martinberlin** [reply]
Can you please share your displays.c entry for this display?
Would be really cool if you can also add the datasheet.
Try also to make the clock slower, like half slower, and see if it makes any difference

**Reply by hongchenfeng** [reply]
ok，waite

**Reply by hongchenfeng** [reply]
const EpdDisplay_t ED080XC1 = {
    .width = 1024,
    .height = 768,
    .bus_width = 8,
    .bus_speed = 10,
    .default_waveform = &epdiy_ED097TC2,
    .display_type = DISPLAY_TYPE_GENERIC,
};

**Reply by hongchenfeng** [reply]
![ED080XC1_Ver1.0_20081201_201404153965.pdf](https://github.com/user-attachments/files/18053050/ED080XC1_Ver1.0_20081201_201404153965.pdf)

**Reply by hongchenfeng** [reply]
slower speed，but no difference. there is display.c and data sheet.but file upload  has mistake.

**Comment by martinberlin** [comment]
Compared this with the ED060XC3 (EINK, same resolution)
and I don't see any noticeable difference. Did you tried all the 6" waveforms just to see if one of them works?
![Screenshot from 2024-12-08 19-45-22](https://github.com/user-attachments/assets/b0e86ffb-44f6-4ebc-8657-caefbf293634)
[ED060XC3 SPEC V1.0.pdf](https://github.com/user-attachments/files/18053567/ED060XC3.SPEC.V1.0.pdf)

**Reply by hongchenfeng** [reply]
ed060xc3 waveform，i tried it firstly, i also thought it's the same of rsolution, but not worked.

**Comment by martinberlin** [comment]
@vroland maybe you can share some  knowledge about this. I've had similar situation with 6" displays but as I commented up there it was fixed just using a different waveform.
But it really should not be waveform dependant, waveform should just control timings to make the -15 or +15 voltage change the display from one color to the next, but this looks as a corrupted data signal where it’s getting out of sync mid-line and since the same line is transmitted multiple times, it shows the image shifted. Maybe you can share some knowledge about what might be causing this?

**Reply by martinberlin** [reply]
Only different timing I see is that XCL cycle time needs to be min 50 nanoseconds instead of 25 in the 6”

I don’t have this display so it’s not possible for me to test. It would be cool to document what part of LCD initialization could be tweaked to modify timings slightly in order to adapt them to this specific panels
